helm/templates/config-jws-public-keys.yaml

```diff
@@ -0,0 +1,22 @@
+{{- if .Values.simulators }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: sim-jws-public-keys
+  labels:
+    app: sim
+    chart: {{ template "mojaloop-simulator.chart" $ }}
+    release: {{ $.Release.Name }}
+    heritage: {{ $.Release.Service }}
+data:
+  {{- range $name, $customConfig := .Values.simulators }}
+  {{- $config := merge $customConfig $.Values.defaults }}
+  {{- if eq ($config.config.schemeAdapter.env.VALIDATE_INBOUND_JWS | upper) "TRUE" }}
+  {{- if eq $config.config.schemeAdapter.secrets.jws.publicKey "" }}
+    {{- fail (printf "INBOUND_JWS enabled and jws public key not specified for '%s'. You need to specify %s.schemeAdapter.secrets.jws.publicKey." $name $name) }}
+  {{- end }}
+  {{ $name }}: |-
+{{ $config.config.schemeAdapter.secrets.jws.publicKey | indent 4 }}
```

Comment:
Here we are supposed to have jws keys of all the peer fsps, but not the key for this fsp.

Reply:
The cost of excluding the FSP in question:
- considerably increased complexity in the chart
- creating n-1 additional config maps (one for every additional fsp)
- having to figure out how to do this
What benefit would be gained for this cost?

Reply:
agreed @msk- 

@ksatya77 that was what we agreed earlier, we would just put all the pub keys since it is way easier and I can't think of any downside